napari/plugins/__init__.py

```diff
@@ -64,3 +65,16 @@ def _initialize_plugins():
         plugin_manager._initialize()
 
     _install_npe2_themes()
+
+    # FIXME is this the right place to initialize plugin keybindings?
```

Comment:
plugin keybindings should have already been handled elsewhere; they are automatically added when you register an action

Reply:
These replaced the line 390 on _npe2.py, I was trying to move the registration out of the registration time to fix the unregistering, but maybe you are right that it can be kept as is, let me try that

Reply:
oh now I remember why I did this: it is possible that a shortcut is overriden by user in the settings, and moving the registration away from _npe2.py allows that